graphrag/query/api.py

```diff
@@ -252,6 +261,12 @@ async def local_search_streaming(
     vector_store_type = vector_store_args.get("type", VectorStoreType.LanceDB)
 
     _entities = read_indexer_entities(nodes, entities, community_level)
+
+    base_dir = cast(str, root_dir) + "/" + config.storage.base_dir
```

Comment:
I think we should use Pathlib here instead of str concat, as it ill break in Windows due to path format